packages/devextreme/js/__internal/scheduler/tooltip_strategies/m_tooltip_strategy_base.ts

```diff
@@ -14,6 +14,8 @@ const TOOLTIP_APPOINTMENT_ITEM_DELETE_BUTTON_CONTAINER = `${TOOLTIP_APPOINTMENT_
 const TOOLTIP_APPOINTMENT_ITEM_DELETE_BUTTON = `${TOOLTIP_APPOINTMENT_ITEM}-delete-button`;
 
 export class TooltipStrategyBase {
+  protected asyncTemplateDeferredList = new Set<Promise<void>>();
```

Comment:
And let's rename this variable because it's not a list and it's not a deferred list :)
E.g
```suggestion
  protected asyncTemplateRenderedArray: Promise<void>[] = [];
```

Or for set:
```suggestion
  protected asyncTemplateRenderedSet = new Set<Promise<void>>();
```